SUBMISSION_PROCESS_RULES.md

```diff
@@ -0,0 +1,176 @@
+
+Submitters unable to self-fund scoring costs can instead self-report only on the [qualification set of workloads](/RULES.md#qualification-set) that excludes some of the most expensive workloads. Based on this performance on the qualification set, the working group will provide - as funding allows - compute to evaluate and score the most promising submissions. Additionally, we encourage researchers to reach out to the [working group](mailto:algorithms@mlcommons.org) to find potential collaborators with the resources to run larger, more comprehensive experiments for both developing and scoring submissions.
+
+#### Verifying scores
+
+The working group will independently verify the scores of the highest-scoring submissions in each ruleset. Results that have been verified by the working group will be clearly marked on the leaderboard.
+
+### Sampling held-out workloads and hyperparameters
+
+After the submission deadline has passed and all submission code is frozen, the working group will sample a specific instance of held-out workloads from the set of randomized workloads. Additionally, every submission in the external tuning ruleset will receive its specific set of 5x20 hyperparameter values grouped by study. This set of hyperparameter values is sampled from the search space provided by the submitters.
+
+The sampling code for the held-out workloads and the hyperparameters is publicly available (**TODO link to both functions!**). Both sampling functions take as input a random seed, which will be provided by a trusted third party after the submission deadline.
+
+### Leaderboard
+
+The publication of the results will contain two separate leaderboards, one for the self-tuning and one for the external tuning ruleset. All valid submissions will be ranked by the benchmark score, taking into account all workloads, including the held-out ones. The leaderboard will clearly mark scores that were verified by the working group.
+
+## Sprit jury & challenging submissions
+
+The spirit jury will be responsible for deciding whether a submission violates the "spirit of the rules". Submitters may challenge other submissions, i.e. request a review by the spirit jury to determine whether a submission violates the rules of the competition. To challenge a submission, please write an email to <algorithms-chairs@mlcommons.org> with the subject "[Challenge] *submission_name*". The email needs to link to the challenged submission and include a detailed description of why the submission should be reviewed.
```

Comment:
Should this section be a bit less precise to make sure we don't constrain ourselves too much? Or is it fine as is?

Reply:
Ok, maybe it is fine as is. I guess the way it is worded sort of implies that we have to do some full review for any email. 

That said, I have an idea to make it even clearer challenges must have a good reason to get "certiorari". For example, if someone just emails us saying "I challenge X submission for Y reason" but Y isn't a valid reason, we don't necessarily have to do some extensive investigation. I guess the spirit jury can still just make a quick ruling.

How about replacing "Submitters may challenge other submissions, i.e. request a review by the spirit jury to determine whether a submission violates the rules of the competition." with

"Submitters with specific concerns about a particular submission can request a review by the spirit jury to determine whether a submission violates the rules of the competition."

Reply:
I changed the sentence "Submitters may challenge..." to "Submitters with specific concerns...".